ext/civiimport/Civi/Api4/Event/Subscriber/ImportSubscriber.php

```diff
@@ -212,36 +212,26 @@ public static function on_civi_afform_get(GenericHookEvent $event): void {
    * @throws \CRM_Core_Exception
    */
   public static function getImportForms(): array {
-    $cacheKey = 'civiimport_forms_' . \CRM_Core_Config::domainID() . '_' . (int) \CRM_Core_Session::getLoggedInContactID();
-    if (\Civi::cache('metadata')->has($cacheKey)) {
-      return \Civi::cache('metadata')->get($cacheKey);
-    }
     $forms = [];
-    try {
-      $importSearches = SearchDisplay::get()
-        ->addWhere('saved_search_id.name', 'LIKE', 'Import\_Summary\_%')
-        ->addWhere('saved_search_id.expires_date', '>', 'now')
-        ->addSelect('name', 'label')
-        ->execute();
-      foreach ($importSearches as $importSearch) {
-        $userJobID = str_replace('Import_Summary_', '', $importSearch['name']);
-        $forms[$importSearch['name']] = [
-          'name' => $importSearch['name'],
-          'type' => 'search',
-          'title' => $importSearch['label'],
-          'base_module' => E::LONG_NAME,
-          'permission' => 'access CiviCRM',
-          'requires' => ['crmSearchDisplayTable'],
-          'layout' => '<div af-fieldset="">
-  <crm-search-display-table search-name="Import_Summary_' . $userJobID . '" display-name="Import_Summary_' . $userJobID . '">
-</crm-search-display-table></div>',
-        ];
-      }
-    }
-    catch (UnauthorizedException $e) {
-      // No access - return the empty array.
+    $importSearches = SearchDisplay::get(FALSE)
+      ->addWhere('saved_search_id.name', 'LIKE', 'Import\_Summary\_%')
+      ->addWhere('saved_search_id.expires_date', '>', 'now')
+      ->addSelect('name', 'label')
+      ->execute();
+    foreach ($importSearches as $importSearch) {
+      $userJobID = str_replace('Import_Summary_', '', $importSearch['name']);
+      $forms[$importSearch['name']] = [
+        'name' => $importSearch['name'],
+        'type' => 'search',
+        'title' => $importSearch['label'],
+        'base_module' => E::LONG_NAME,
+        'permission' => 'access CiviCRM',
+        'requires' => ['crmSearchDisplayTable'],
+        'layout' => '<div af-fieldset="">
+  <c-search-display-table search-name="Import_Summary_' . $userJobID . '" display-name="Import_Summary_' . $userJobID . '">
```

Comment:
Is this resistant to injection? Should `$userJobID` have `htmlentities()` applied to it?

Reply:
Yeah possibly - I'd rather err on the side of adding that & have to remove it I think - I will push up separately

Reply:
Actually @ufundo has made the point elsewhere that these summary forms are not currently being linked to in the UI - I think the goal was to replace the Import summary page with them but that stalled out as did the effort to embed them in the import summary page - I think I would be Ok with @ufundo proposal to fully remove the code that creates these summary afforms (which also makes the htmlentities concern obsolete)